components/Bubbles/BubblesGame.js

```diff
@@ -27,10 +28,11 @@ const SCALE = {
   height: Dimensions.get('window').height / 800,
 };
 // TODO: do we need offset?
-const OFFSET = 60;
+const OFFSET = 10;
 const GAME_TIME_OUT = 115000;
 const MAX_NUMBER_BUBBLES = 15;
 const FOUTAIN_LOCATION = {top: 0, left: 0};
+const FOUTAIN_SIZE = { width: 270 * SCALE.width, height: 258 * SCALE.height};
```

Comment:
since we pass the in the prop scale.screenWidht/Height let's use that instead. Set this in constructor using this.scale.screenWidth ...